docs/src/add-services/mysql/_index.md

```diff
@@ -348,6 +348,31 @@ ALTER TABLE table_name CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_
 
 For further details, see the [MariaDB documentation](https://mariadb.com/kb/en/character-set-and-collation-overview/).
 
+## Password generation
+ 
+When you create an endpoint to allow your app and database to communicate,
+an empty password is generated by default.
+If you add multiple endpoints,
+having empty passwords can cause issues with your app.
+ 
+To solve this,
+define custom endpoints in your [service configuration](#configuration-options).
+For each custom endpoint you create, 
+you get an automatically generated password,
+similarly to when you create [multiple databases](#multiple-databases).
+Note that you can't customize these automatically generated passwords.
+
+After your custom endpoints are exposed as relationships in your [app configuration](../../create-apps/_index.md),
+you can retrieve the password for each endpoint
+through the `PLATFORM_RELATIONSHIPS` [environment variable](../../development/variables/use-variables.md#use-platformsh-provided-variables).
+You can access the `PLATFORM_RELATIONSHIPS` environment variable [in a shell](../../development/variables/use-variables.md#access-variables-in-a-shell)
```

Comment:
:question: Why would you want to do this in this case? What's the benefit of pushing people toward this option as the first one?